src/lib/scale/create.tsx

```diff
@@ -307,8 +328,24 @@ $ sf scale create -n 8 --horizon '30m'
   )
   .option("-y, --yes", "Automatically confirm the command.")
   .action((options) => {
+    const colocationStrategy = options.cluster
+      ? { type: "pinned" as const, cluster_name: options.cluster }
+      : match(options.colocationStrategy)
+        .with("anywhere", () => ({ type: "anywhere" as const }))
+        .with("colocate", () => ({ type: "colocate" as const }))
+        .with("colocate-pinned", () => ({ type: "colocate-pinned" as const }))
+        .with(
+          "pinned",
+          () =>
+            logAndQuit("Invalid colocation strategy: `-c`/`--cluster` not set"),
+        )
```

Comment:
This seems to duplicate `parseColocationStrategy `

Reply:
not the same, because it involves -c flag